tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -16,130 +17,138 @@
-        return self.response
+def mocked_requests_post_404(*args, **kwargs):
+    return MockResponse({}, 404)
 
-    def close(self):
-        print("close MockConnection")
+def mocked_requests_httperror_exception():
+    return requests.exceptions.HTTPError()
 
 @pytest.fixture
-def missing_domain_env_param(monkeypatch):    
-    monkeypatch.setenv('vetext_api_endpoint_path', "/some/path")
+def missing_domain_env_param(monkeypatch):
+    monkeypatch.setenv('vetext_api_endpoint_path', vetext_uri_path)
     monkeypatch.setenv('vetext_api_auth_ssm_path', 'ssm')
 
+
 @pytest.fixture
-def missing_api_endpoint_path_env_param(monkeypatch):    
-    monkeypatch.setenv('vetext_api_endpoint_domain', "some.domain")
+def missing_api_endpoint_path_env_param(monkeypatch):
+    monkeypatch.setenv('vetext_api_endpoint_domain', vetext_domain)
     monkeypatch.setenv('vetext_api_auth_ssm_path', 'ssm')
 
+
 @pytest.fixture
-def missing_ssm_path_env_param(monkeypatch):    
-    monkeypatch.setenv('vetext_api_endpoint_domain', "some.domain")
-    monkeypatch.setenv('vetext_api_endpoint_path', "/some/path")
+def missing_ssm_path_env_param(monkeypatch):
+    monkeypatch.setenv('vetext_api_endpoint_domain', vetext_domain)
+    monkeypatch.setenv('vetext_api_endpoint_path', vetext_uri_path)
+
 
 @pytest.fixture
-def all_path_env_param_set(monkeypatch):    
-    monkeypatch.setenv('vetext_api_endpoint_domain', "some.domain")
-    monkeypatch.setenv('vetext_api_endpoint_path', "/some/path")
+def all_path_env_param_set(monkeypatch):
+    monkeypatch.setenv('vetext_api_endpoint_domain', vetext_domain)
+    monkeypatch.setenv('vetext_api_endpoint_path', vetext_uri_path)
     monkeypatch.setenv('vetext_api_auth_ssm_path', 'ssm')
     monkeypatch.setenv('vetext_request_drop_sqs_url', "someurl")
     monkeypatch.setenv('vetext_request_dead_letter_sqs_url', "someurl")
 
+
 LAMBDA_MODULE = "lambda_functions.vetext_incoming_forwarder_lambda.vetext_incoming_forwarder_lambda"
 
+
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn)])
 def test_verify_parsing_of_twilio_message(event):
     response = process_body_from_alb_invocation(event)
 
     assert response, "The dictionary should not be empty"
     assert 'AddOns' not in response
 
+
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
-def test_request_makes_vetext_call(mocker, all_path_env_param_set, http_success_response, event):
+def test_request_makes_vetext_call(mocker, all_path_env_param_set, event):
     sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_retry_sqs')
-    mocker.patch(f'{LAMBDA_MODULE}.read_from_ssm', return_value="ssm")    
-    mocker.patch("http.client.HTTPSConnection", return_value=MockConnection(http_success_response))    
+    mocker.patch(f'{LAMBDA_MODULE}.read_from_ssm', return_value="ssm")
+    mocker.patch(f'{LAMBDA_MODULE}.requests.post',
+                 side_effect=mocked_requests_post_success)
```

Comment:
requests.post returns a response object.  Should this be `retrurn_value` instead of `side_effect`, as you were doing before?  Ditto for the same use of `side_effect` elsewhere.